Platform.js

```diff
@@ -55,542 +55,542 @@ var fElapsed = 0;
 
 var colors = [color(255, 158, 158), color(173, 185, 255), color(169, 255, 158), color(181, 181, 181), color(254, 255, 168), color(252, 154, 239), color(255, 219, 173)];
 var turnLeft = function(start, end, size){
-    return min(abs(start-end+size-1), min(abs(start-end-1), abs(start-end-size-1))) < min(abs(start-end+size+1), min(abs(start-end+1), abs(start-end-size+1)));
+	return min(abs(start-end+size-1), min(abs(start-end-1), abs(start-end-size-1))) < min(abs(start-end+size+1), min(abs(start-end+1), abs(start-end-size+1)));
 };
-var Entity = function(x, y, ID, flag){
-    this.RAM = [];
-    this.a = 0;  // "actual" angle
-    this.acceleration = 0.05;
-    this.alignment = ID;
-    this.alive = true;
-    this.cd = 0;
-    this.discriminator = discrim++;
-    this.fC = 0; // flag color & id (if holding one)
-    this.fatigue = 90;
-    this.flag = !!flag;
-    this.fov = 120;
-    this.hasFlag = false;
-    this.lookCD = 0;
-    this.max = 1000;
-    this.msgCD = 0;
-    this.regen = 0;
-    this.restart = 100;
-    this.safe = true;
-    this.stamina = 0.4*this.max;
-    this.tA = 0; // target angle
-    this.temp = [];
-    this.topSpeed = 3;
-    this.v = 0;
-    this._v = 0;
-    this.x = x;
-    this.y = y;
+var Entity = function(x, y, team, flag){
+	this.RAM = [];
+	this.angle = 0; 
+	this.acceleration = 0.05;
+	this.teamWith = team;
+	this.alive = true;
+	this.CD = 0;
+	this.ID = IDs++;
+	this.flagInfo = 0; // flag color & id (if holding one)
+	this.fatigue = 90;
+	this.isFlag = !!flag;
+	this.FOV = 120;
+	this.hasFlag = false;
+	this.lookCD = 0;
+	this.max = 1000;
+	this.msgCD = 0;
+	this.regen = 0;
+	this.restart = 100;
+	this.safe = true;
+	this.stamina = 0.4*this.max;
+	this.targetAngle = 0; // target angle
+	this.temp = [];
+	this.topSpeed = 3;
+	this.currentSpeed = 0;
+	this.targetSpeed = 0;
```

Comment:
re-alphabetize

Reply:
oh i never noticed you have the variables alphatized